.github/workflows/linters.yml

```diff
@@ -0,0 +1,50 @@
+name: Linters
+
+on: pull_request
```

Comment:
- [ ] Your Pull Request is empty and does not describe the feature(s) you want to merge. Please add a descriptive Pull Request title and summary describing the features.